test/e2e/tests/tokens/token-sort.spec.ts

```diff
@@ -3,102 +3,66 @@ import { Context } from 'mocha';
     ganacheOptions: {
       ...defaultGanacheOptions,
-      chainId: parseInt(chainId, 16),
+      chainId: parseInt(mainnetChainId, 16),
     },
   };
 
-  const importToken = async (driver: Driver) => {
-    await driver.clickElement(`[data-testid="import-token-button"]`);
-    await driver.clickElement(`[data-testid="importTokens"]`);
-    await clickNestedButton(driver, 'Custom token');
-    await driver.fill(
-      '[data-testid="import-tokens-modal-custom-address"]',
-      tokenAddress,
-    );
-    await driver.waitForSelector('p.mm-box--color-error-default');
-    await driver.fill(
-      '[data-testid="import-tokens-modal-custom-symbol"]',
-      symbol,
-    );
-    await driver.delay(2000);
-    await driver.clickElement({ text: 'Next', tag: 'button' });
-    await driver.clickElement(
-      '[data-testid="import-tokens-modal-import-button"]',
-    );
-    await driver.findElement({ text: 'Token imported', tag: 'h6' });
-  };
-
-  it('should sort alphabetically and by decreasing balance', async function () {
+  it('should sort tokens alphabetically and by decreasing balance', async function () {
     await withFixtures(
       {
-        ...fixtures,
+        ...testFixtures,
         title: (this as Context).test?.fullTitle(),
       },
       async ({ driver }: { driver: Driver }) => {
         await unlockWallet(driver);
-        await importToken(driver);
-
-        const tokenListBeforeSorting = await driver.findElements(
-          '[data-testid="multichain-token-list-button"]',
-        );
-        const tokenSymbolsBeforeSorting = await Promise.all(
-          tokenListBeforeSorting.map(async (tokenElement) => {
-            return tokenElement.getText();
-          }),
-        );
-
-        assert.ok(tokenSymbolsBeforeSorting[0].includes('Ethereum'));
 
-        await driver.clickElement('[data-testid="sort-by-popover-toggle"]');
-        await driver.clickElement('[data-testid="sortByAlphabetically"]');
+        const homePage = new HomePage(driver);
+        const assetListPage = new AssetListPage(driver);
 
-        await driver.delay(regularDelayMs);
-        const tokenListAfterSortingAlphabetically = await driver.findElements(
-          '[data-testid="multichain-token-list-button"]',
-        );
-        const tokenListSymbolsAfterSortingAlphabetically = await Promise.all(
-          tokenListAfterSortingAlphabetically.map(async (tokenElement) => {
-            return tokenElement.getText();
-          }),
+        await homePage.check_pageIsLoaded();
+        await assetListPage.importCustomToken(
+          customTokenAddress,
+          customTokenSymbol,
         );
 
-        assert.ok(
-          tokenListSymbolsAfterSortingAlphabetically[0].includes('ABC'),
-        );
+        const initialTokenList = await assetListPage.getTokenListNames();
+        assert.ok(initialTokenList[0].includes('Ethereum'));
+        await assetListPage.sortTokenList('alphabetically');
```

Comment:
maybe this check could be a method inside the `asset-list` class, something like:

`assetListPage.isTokenInAssetsList(tokenName)`

So it can be utilized in more specs if that's needed. What do you think?